### PR TITLE
updpatch: update to linux 6.6.1.arch1

### PR DESCRIPTION
**Pull request by CoelacanthusHex**
- enable some VisionFive2 driver
- drop old disable-sv57 patch because 6.6 include better way

### CODE REVIEW

File: linux/riscv64.config-patch [hunk 1]
```diff
@@ -773,15 +780,23 @@
...
 +# CONFIG_ARM_SUN8I_A33_MBUS_DEVFREQ is not set
  CONFIG_PM_DEVFREQ_EVENT=y
  CONFIG_EXTCON=y
  
-@@ -9057,8 +9262,10 @@
+@@ -8609,6 +8820,7 @@
+ CONFIG_PALMAS_GPADC=m
+ CONFIG_RICHTEK_RTQ6056=m
+ CONFIG_SD_ADC_MODULATOR=m
++# CONFIG_SUN20I_GPADC is not set
```
Comment (felixonmars) [inline linux/riscv64.config-patch:795]: Looks like this is a driver for D1, should we enable it?

File: linux/riscv64.patch [hunk 2]
```diff
@@ -1,87 +0,0 @@
-diff --git PKGBUILD PKGBUILD
-index 2b0f6d9..b1b9999 100644
---- PKGBUILD
-+++ PKGBUILD
-@@ -31,6 +31,8 @@ source=(
-   https://cdn.kernel.org/pub/linux/kernel/v${pkgver%%.*}.x/${_srcname}.tar.{xz,sign}
-   $url/releases/download/$_srctag/linux-$_srctag.patch.zst{,.sig}
-   config  # the main kernel config file
-+  riscv64.config-patch
```
Comment (felixonmars) [inline linux/riscv64.patch:9]: I assume the other changes are still needed...

**Review by felixonmars — CHANGES_REQUESTED** [review]

File: linux/riscv64.config-patch (hunk 1 above)
Comment (CoelacanthusHex) [reply linux/riscv64.config-patch]: done

File: linux/riscv64.patch (hunk 2 above)
Comment (CoelacanthusHex) [reply linux/riscv64.patch:9]: done

**Pull request merged**